packages/flutter/lib/src/rendering/debug.dart

```diff
@@ -5,7 +5,7 @@
 import 'package:flutter/painting.dart';
 import 'package:flutter/rendering.dart';
 
-export 'package:flutter/services.dart' show debugPrint;
+export 'package:flutter/services.dart';
```

Comment:
I don't understand this change.  Previously we were exporting only a single name from services.dart.  Now we're exporting the whole thing.